CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
```

Comment:
@stonko1994 Added this changelog since it seems like we did not have one previously. Does this match what we have for other platforms?

Reply:
yes looks good 🙂